src/routes/search/components/SearchActiveHeader.js

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
Maybe rather `SearchHeader`? What does the `active` part mean?

Reply:
Sounds good. Active was meant to signalize that there is an active search and not just a search icon.